core/src/main/scala/org/apache/spark/ui/jobs/AllJobsPage.scala

```diff
@@ -172,6 +172,7 @@ private[ui] class AllJobsPage(parent: JobsTab) extends WebUIPage("") {
 
     val jobEventJsonAsStrSeq = makeJobEvent(jobs)
     val executorEventJsonAsStrSeq = makeExecutorEvent(executors)
+    val offset = TimeZone.getDefault().getOffset(System.currentTimeMillis()) / 1000 / 60
```

Comment:
Ur, I think you might miss my previous comment about this . Or, did I really miss something?
> Since getRawOffset does not consider Daylight Saving Time, I needed to use the above.